sys/linux/dev_msr.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+#include <asm/msr.h>
+#include <uapi/asm/msr.h>
+#include <uapi/linux/fcntl.h>
+
+resource fd_msr[fd]
+
+syz_open_dev$MSR(dev ptr[in, string["/dev/cpu/#/msr"]], id intptr, flags const[O_RDONLY]) fd_msr
+read$msr(fd fd_msr, buf buffer[out], count len[buf])
+# TODO: We cannot write arbitrary data
+# write$msr(fd fd_msr, buf buffer[in], count len[buf])
+
+ioctl$X86_IOC_RDMSR_REGS(fd fd_msr, cmd const[X86_IOC_RDMSR_REGS], arg ptr[in, array[int32, 8]])
```

Comment:
The neutralize logic also uses X86_IOC_WRMSR_REGS, so we need:
```
_ = X86_IOC_WRMSR_REGS
```
so that the value is pulled into the .const file.